builder/ConfTargets.cmake

```diff
@@ -106,6 +106,7 @@ cmake_dependent_option(
 option( MFX_ENABLE_VP9_VIDEO_ENCODE "Enable VP9 encoder?" ON)
 
 option( MFX_ENABLE_ASC "Enable ASC support?"  ON )
+option( MFX_ENABLE_LOW_DELAY_BS_OUT "Enable Partial Bitstream optput support?"  OFF )
```

Comment:
Should we really have dedicated command line option for that? You see, cmdline cmake options were introduced to switch on/off big features like entire codec or enable/disable EU shaders. This particular new feature does not seem to imply any new EU shader, does it? Can you, please, hint how it will be implemented? If it will be handled on mediasdk .cpp level and code won't be dramatically big, I don't think we need a special switch for it on cmdline level, just ever-on define in config.h will be enough.

Reply:
Agree, will remove this option

Reply:
As of now it is not removed.

Reply:
done